test/common/performance/sender/http_load_generator.go

```diff
@@ -124,6 +132,7 @@ func NewHttpLoadGeneratorFactory(sinkUrl string, minWorkers uint64) LoadGenerato
 				},
 			}}),
 			vegeta.Workers(minWorkers),
+			vegeta.MaxBody(0),
```

Comment:
It looks `-1` means no limit, does `0` means no body content is allowed? Can we have a comment here?

Reply:
Yes there is no need to read the body for our test. With 0 the body is just discarded and not loaded in memory